.github/CODEOWNERS

```diff
@@ -59,6 +59,7 @@
 # TODO: if we ever add `.internal` API reports, we will want to omit them here
 /**/api-report/*.api.md                        @microsoft/fluid-cr-api
```

Comment:
If the idea of "alpha" is to move fast with ideas and try things out, maybe instead we should only require API review for *.public.api.md?  And maybe *.beta.api.md, *.legacy.alpha.api.md?

Reply:
I like this. New proposal.

Reply:
the thing to keep in mind is that this could result in a heavy lift where substantial changes are requested from alpha to beta, as we are essentially saying review is deferred until the beta phase.

Reply:
it would also be useful if the use case could be expanded upon in this review? For instance, what is the plan, and what about the plan makes the current state hard to deal with? Is there a possibility to work on this as internal, and only switch to alpha before release?

Reply:
I would say that review enforcement is deferred until beta. Hopefully APIs are getting reviewed once there is something worthy. Maybe that is alpha maybe that is later.
I certainly welcome and encourage looking at the presence API, but that doesn't need to happen at PR time.
I defer to API council for policy they want.